src/drivers/driver_zephyr.c

```diff
@@ -1997,6 +2352,35 @@ int wpa_drv_zep_send_mlme(void *priv, const u8 *data, size_t data_len, int noack
 	return ret;
 }
 
+#ifdef CONFIG_WIFI_NM_HOSTAPD_AP
+int wpa_drv_hapd_send_eapol(void *priv, const u8 *addr, const u8 *data, size_t data_len,
```

Comment:
If there are no changes b/w the two variants, do we need the `#ifdef`, can't we reuse the same function?

Reply:
Updated. Moved `#ifdef` into the function since data structures are different between wpa_supplicant and hostapd.